### ISSUE TITLE
PermissionError: [WinError 5] Access is denied

### ISSUE THREAD

**Issue by hugomrdias** [jisaacks/GitGutter#103]
i get this error running ST3 on Windows 7

``` bash
Traceback (most recent call last):
  File "C:\Program Files\Sublime Text 3\sublime_plugin.py", line 526, in run_
    return self.run()
  File "C:\Users\Hugo\AppData\Roaming\Sublime Text 3\Installed Packages\GitGutter.sublime-package\git_gutter.py", line 34, in run
  File "C:\Users\Hugo\AppData\Roaming\Sublime Text 3\Installed Packages\GitGutter.sublime-package\view_collection.py", line 36, in diff
  File "C:\Users\Hugo\AppData\Roaming\Sublime Text 3\Installed Packages\GitGutter.sublime-package\git_gutter_handler.py", line 150, in diff
  File "C:\Users\Hugo\AppData\Roaming\Sublime Text 3\Installed Packages\GitGutter.sublime-package\git_gutter_handler.py", line 166, in run_command
  File "X/subprocess.py", line 818, in __init__
  File "X/subprocess.py", line 1096, in _execute_child
PermissionError: [WinError 5] Access is denied
```

**Comment by rushfly** [comment]
First, I removed gitgutter through package control.
Then  deleted all files in the directory of "Data\Packages\User\Package Control.cache" after close sublime.
Run again and reinstall gitgutter, it's works!

**Comment by jisaacks** [comment]
@rushfly thanks for the update. Can someone else confirm this solves it for them too?

**Comment by geedew** [comment]
@jisaacks Confirmed that it does not work (removing and wiping cache).

This continues for me on Windows 10, ST3

**Comment by hsandt** [comment]
I solved the problem by changing my GitGutter User preferences. In my case, the git binary path was wrong, not including the filename itself. It was set to `C:\Program Files (x86)\Git\bin` instead of `C:\Program Files (x86)\Git\bin\git.exe`.

Open User\GitGutter.sublime-settings (Preferences > Package Settings > GitGutter > Settings - User):
```
{
  "git_binary": "C:\\Program Files (x86)\\Git\\bin\\git.exe"  // replace with your own path, including .exe
}
```

Long story:

After looking at the source code (`git_gutter_handler.py`), I searched on issues with Python subprocess.Popen and found a fix on http://stackoverflow.com/questions/3005437/windowserror-error-5-access-is-denied, some answers recalling you to call subprocess.Popen with the executable full path. By printing the arguments of Popen I realized that GitGutter really needs you to define the full executable path of git.

https://github.com/jisaacks/GitGutter/issues/191 and https://github.com/jisaacks/GitGutter/issues/178#issuecomment-72684981 also refer to the git_binary setting, used as argument for Popen.
